DSWaveformImage/DSWaveformImage/Common/WaveformImageDrawer+macOS.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import AVFoundation
+import AppKit
+import CoreGraphics
+
```

Comment:
Keep this in the main framework and conditional it on `#if os(macOS)` and the iOS version based on #if os(iOS) and you're cruisin'!